cuesubmit/cuesubmit/Submission.py

```diff
@@ -18,11 +18,12 @@
 from __future__ import absolute_import
 
 from builtins import str
-from outline import Outline, cuerun
-from outline.modules.shell import Shell
 
 from cuesubmit import Constants
 from cuesubmit import JobTypes
+import outline.cuerun
+import outline.loader
+import outline.modules.shell
```

Comment:
What's the reasoning behind changing the imports?
Having to use `outline.modules.shell` everywhere decreases the readability quite a bit.

Reply:
Back when I was doing the Python 2 -> 3 migration, I seem to remember this style caused the least amount of issues in all python versions. Can't remember the exact details though.

In general, I do prefer it -- it makes the code more readable for me, as it's extremely clear what classes we're using and where they're coming from. If you're finding it problematic here though, I don't feel that strongly about it and can change it back, as long as it passes tests in Python2/3 and lint.